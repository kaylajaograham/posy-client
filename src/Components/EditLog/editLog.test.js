import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EditLogForm from './editLog';

describe(`edLog component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders a form.editLog by default', () => {
    const wrapper = shallow(<EditLogForm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the editLog given props', () => {
    const wrapper = shallow(<EditLogForm {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})