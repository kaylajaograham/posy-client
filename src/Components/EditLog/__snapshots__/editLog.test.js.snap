// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`edLog component renders a form.editLog by default 1`] = `
<form
  className="editLogForm"
  onSubmit={[Function]}
>
  <div
    className="editLog"
  >
    <h3>
      Edit Log
    </h3>
    <div
      className="log_name"
    >
      <label
        htmlFor="editLogForm__log_name"
      >
        Title
      </label>
      <input
        id="editLogForm__log_name"
        name="log_name"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="log_entry"
    >
      <label
        htmlFor="editLogForm__log_entry"
      >
        Entry
      </label>
      <textarea
        id="editLogForm__log_entry"
        name="log_entry"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <button
      className="button"
      type="submit"
    >
      Save
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Go Back
    </button>
  </div>
</form>
`;

exports[`edLog component renders the editLog given props 1`] = `
<form
  className="editLogForm"
  onSubmit={[Function]}
>
  <div
    className="editLog"
  >
    <h3>
      Edit Log
    </h3>
    <div
      className="log_name"
    >
      <label
        htmlFor="editLogForm__log_name"
      >
        Title
      </label>
      <input
        id="editLogForm__log_name"
        name="log_name"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="log_entry"
    >
      <label
        htmlFor="editLogForm__log_entry"
      >
        Entry
      </label>
      <textarea
        id="editLogForm__log_entry"
        name="log_entry"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <button
      className="button"
      type="submit"
    >
      Save
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Go Back
    </button>
  </div>
</form>
`;
